packages/insight-previous/package.json

```diff
@@ -65,7 +65,7 @@
     "prettier": "^1.14.3",
     "rxjs": "5.5.12",
     "sw-toolbox": "3.6.0",
-    
+    "winston": "^3.1.0",
```

Comment:
No additional packages is necessary here. Please, check Copay how it handles the console.log. https://github.com/bitpay/copay/blob/master/src/providers/logger/logger.ts

Reply:
Thank you for the reference! Removed winston and matched copay logger 👍